ckanext/nhs/templates/home/snippets/latest_info.html

```diff
@@ -1,4 +1,4 @@
-{% set resources= h.get_latest_resources() %}
+{% set resources= [] %}
```

Comment:
Why are we removing the latest resources from the latest_info?

Reply:
good catch @MuhammadIsmailShahzad. This is not supposed to merge in the main branch. I specifically changed it for my local setup cuz it throwing some errors.  i will update the PR and let you know.

Reply:
ok. I can see its been fixed in https://github.com/datopian/nhs/pull/10/commits/c6079394c57cad2cc50f2b121f6c5e7b7b5c638f.